src/vivarium_csu_hypertension_sdc/components/hypertension_intervention.py

```diff
@@ -0,0 +1,159 @@
+from scipy import stats
+
+import numpy as np
+import pandas as pd
+
+from vivarium_csu_hypertension_sdc.components import utilities
+from vivarium_csu_hypertension_sdc.components.globals import (DOSAGE_COLUMNS, HYPERTENSIVE_CONTROLLED_THRESHOLD)
+
+
+class TreatmentAlgorithm:
+    configuration_defaults = {
+        'hypertension_treatment': {
+            'high_systolic_blood_pressure_measurement': {
+                'error_sd': 6,
+            },
+            'therapeutic_inertia': {
+                'mean': 0.136,
+                'sd': 0.0136,
+            },
+            'adverse_events': {
```

Comment:
No adverse events in this model, I think.

Reply:
hmm ok - these values are from the spreadsheet they sent so I threw them in but that's certainly easier if we don't use them